types/reactstrap/lib/Button.d.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { CSSModule } from '../index';
 
-export type ButtonProps<T = {}> = React.HTMLProps<HTMLButtonElement> & {
+export type ButtonProps<T = {}> = React.AllHTMLAttributes<HTMLButtonElement> & {
```

Comment:
Whoops, thought I caught all of the `HTMLProps` that were relevant when I did the generic extension. I don't have time to test it myself personally, but does `ButtonHTMLAttributes` fix your issue instead of `AllHTMLAttributes` since that's a better fit?

Reply:
@FaithForHumans Yes, it seems to work well :) You want me to submit a new PR? Also, should I increment the version as well?

Reply:
Instead of submitting a new PR, just push the change to this existing PR.

Don't increment the version since it's just a bug fix (the DefinitelyTyped backed auto-increments the patch when it gets published).